IPJ/IPJ6_PictureRGB256/src/main/java/PictureRGB256.java

```diff
@@ -0,0 +1,41 @@
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.*;
+import java.io.IOException;
+
+public class PictureRGB256 {
+    final int width = 256;
+    final int height = 256;
+    final String error = "error!!!";
+    final String fileFormat = "jpg";
+    final String pathName = "//home//lukasz//Pulpit//matrixToFile256pxl.jpg";
```

Comment:
Same with file. It would be nice if program would be elastic

Reply:
@MatDls as above comment